client/ayon_core/tools/utils/folders_widget.py

```diff
@@ -508,6 +562,12 @@ def _get_selected_index(self):
 
     def _on_project_selection_change(self, event):
         project_name = event["project_name"]
+        assigned_folder_items = self._controller.get_assigned_folder_items(project_name)
```

Comment:
I'm realizing there's a chance that function would return `None` so the `for` loop would error out, let me indent it in a conditional

Reply:
done at https://github.com/ynput/ayon-core/pull/622/commits/d640dd1b274db0ff9f663ee500e2336bca96a811